packages/core/src/interpreter.ts

```diff
@@ -123,7 +124,7 @@ export class Interpreter<
 
   public src?: string;
 
-  public root: any;
+  public root?: AnyActorRef;
```

Comment:
```suggestion
  public root: AnyActorRef;
```

An actor system will always have a root actor ref

Reply:
@davidkpiano I'm not getting any errors if I omit this entirely.